packages/create-atomic/project.json

```
@@ -1,7 +1,18 @@
 {
   "name": "create-atomic",
-  "root": "packages/ui/atomic/create-atomic",
-  "sourceRoot": "packages/ui/atomic/create-atomic/src",
-  "projectType": "application",
-  "implicitDependencies": ["create-atomic-template"]
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
```

We need the implict dependencies so everything build in order.
```suggestion
  "$schema": "../../node_modules/nx/schemas/project-schema.json",
  "implicitDependencies": ["create-atomic-template"],
```